src/simperium/channel.js

```diff
@@ -237,6 +243,11 @@ internal.handleChangeError = function( err, change, acknowledged ) {
 				this.localQueue.dequeueChangesFor( change.id );
 			}
 
+			break;
+		case CODE_DUPLICATE_CHANGE:
+			// no need to do anything else here - we already got this
+			this.localQueue.dequeueChangesFor( change.id );
```

Comment:
I believe this will remove more changes than we want it to. A `409` should mean that the `LocalQueue.sent[object.id]` change needs to be "cleared" since the server has already seen it.

However the `LocalQueue.queues[object.id]` may have changes that need to send once the in flight change has been acknowledged.

`dequeueChangesFor` will remove both the `.sent[object.id]` (in flight change) and the `.queue[object.id]` (list of changes waiting for the in-flight change to be acknowledged).